autogen/agentchat/conversable_agent.py

```diff
@@ -282,13 +285,26 @@ def _message_to_dict(message: Union[Dict, str]) -> Dict:
         else:
             return dict(message)
 
+    @staticmethod
+    def _normalize_name(name):
```

Comment:
If we modify the function name under the hood users may be surprised. Perhaps we only validate instead of doing modification.

Reply:
This is used for LLM responses, not for user input. User input uses `_assert_valid_name` below

Reply:
will add some comments

Reply:
Okay got it. Can you add a comment explaining it? Helpful for future refactoring.